api/python/v2/index.md

```diff
@@ -126,7 +128,7 @@ The Python client library sends HTTP POST requests to the Data Commons [REST API
 | API | Endpoint | Description | Response type |
 | --- | --- -----| ----------- |---------------|
 | Observation | [`observation`](observation.md) | Fetches statistical observations (time series) | `ObservationResponse` |
-| [Observations Pandas DataFrame](observation_pandas.md) | Similar to the `fetch_observatons_by_entity_dcids`|`type` methods of the Observaton endpoint, except that the functonality is provided by a single method of the `DataCommonsClient` class directly, instead of an intermediate endpoint. Requires the optional `Pandas` module. | `pd.DataFrame` |
+| [Observations Pandas DataFrame](pandas.md) | Similar to the `fetch_observatons_by_entity_dcids` and `fetch_observations_by_entity_type` methods of the Observaton endpoint, except that the functionality is provided by a single method of the `DataCommonsClient` class directly, instead of an intermediate endpoint. Requires the optional `Pandas` module. | `pd.DataFrame` |
```

Comment:
```suggestion
| [Observations Pandas DataFrame](pandas.md) | Similar to the `fetch_observatons_by_entity_dcids` and `fetch_observations_by_entity_type` methods of the Observation endpoint, except that the functionality is provided by a single method of the `DataCommonsClient` class directly, instead of an intermediate endpoint. Requires the optional `Pandas` module. | `pd.DataFrame` |
```